<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1169</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>CPU</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="3" column="0">
         <widget class="QLineEdit" name="lineEdit_Cmd"/>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSplitter" name="splitter_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QSplitter" name="splitter">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <widget class="QTableWidget" name="tableWidget_Disassembly">
                 <attribute name="horizontalHeaderCascadingSectionResizes">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>200</number>
                 </attribute>
                 <column>
                  <property name="text">
                   <string>地址</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>机器码</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>反汇编</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>符号</string>
                  </property>
                 </column>
                </widget>
                <widget class="QTableWidget" name="tableWidget_Memory">
                 <property name="showGrid">
                  <bool>true</bool>
                 </property>
                 <property name="columnCount">
                  <number>2</number>
                 </property>
                 <attribute name="horizontalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="horizontalHeaderCascadingSectionResizes">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>300</number>
                 </attribute>
                 <column/>
                 <column/>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QSplitter" name="splitter_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QFrame" name="frame_2">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QSplitter" name="splitter_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <widget class="QTableWidget" name="tableWidget_Register">
                  <property name="maximumSize">
                   <size>
                    <width>596</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="columnCount">
                   <number>2</number>
                  </property>
                  <attribute name="horizontalHeaderVisible">
                   <bool>false</bool>
                  </attribute>
                  <row>
                   <property name="text">
                    <string>EAX</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>EBX</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>ECX</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>EDX</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>EBP</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>ESP</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>ESI</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>EDI</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>EIP</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>EFLAG</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string/>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>ZF</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>PF</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>AF</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>OF</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>SF</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>DF</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>CF</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>TF</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>IF</string>
                   </property>
                  </row>
                  <column/>
                  <column/>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QFrame" name="frame_3">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QTableWidget" name="tableWidget_Stack"/>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>日志</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>断点</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_BpTable">
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <column>
           <property name="text">
            <string>类型</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>地址</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>长度</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>DLL</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="1">
         <widget class="QTableWidget" name="tableWidget_FUC">
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <column>
           <property name="text">
            <string>函数名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>地址</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_DLL">
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <column>
           <property name="text">
            <string>DLL名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>地址</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1169</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>调试</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>附加</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>步进</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>步过</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
